src/Application/Actions/Donations/Confirm.php

```diff
@@ -115,16 +114,25 @@ protected function action(Request $request, Response $response, array $args): Re
                 paymentIntentId: $paymentIntentId,
             );
         } catch (ApiErrorException $exception) {
-            $this->logger->error(sprintf(
-                'Stripe %s on Confirm updatePaymentMethodBillingDetail for donation %s (%s): %s',
-                get_class($exception),
-                $donation->getUuid(),
-                $paymentIntentId,
-                $exception->getMessage(),
-            ));
-
```

Comment:
removing this manual error log - it's not needed, and causes double errors, since throw an exception on line 128